docs/features/dynamic-analysis.work.md

```diff
@@ -0,0 +1,14 @@
+Dynamic Analysis
+===============
+
+This feature enables instrumentation of binaries to collect information for dynamic analysis.
+
+--------------------
+
+TODO:
+
+- Generate code to compute MVIDs less often than once per invoked method.
```

Comment:
I'll need to learn how to write markdown, and will need to reconstruct the history of what's been done, so that will be in a follow-on PR.